kmir/src/tests/integration/test-data/prove-rs/simple-spec.k

```diff
@@ -63,29 +61,28 @@ module SIMPLE-SPEC
           <function>
             <fnKey> FN_KEY </fnKey>
               <localDecl>
-                <index> INDEX1 </index>
+                <index> PLACE1 </index>
                 <value> _ => 8 </value>
                 ...
               </localDecl>
               <localDecl>
-                <index> INDEX2 </index>
+                <index> PLACE2 </index>
                 <value> _ => 9 </value>
                 ...
               </localDecl>
               ...
           </function>
-    requires INDEX1 ==Int Local2Int(PLACE1) andBool INDEX2 ==Int Local2Int(PLACE2) andBool INDEX1 =/=Int INDEX2
+    requires PLACE1 =/=K PLACE2
```

Comment:
This change will mean that we have more `==K` and `=/=K`

Reply:
It looks good to me.
Furthermore, will it be a good idea to extend the builtin types with `Local`, which is to implement a `==Local`, `=/=Local` and other operations.

It doesnot seem immediately necessary though. What do you think?

Reply:
I was having this thought as well. I think for now we can leave it as `==K`, but there may be a point in the future where a change is required.